Formula/a/antibody.rb

```diff
@@ -23,7 +23,7 @@ class Antibody < Formula
   depends_on "go@1.17" => :build
 
   def install
-    system "go", "build", "-ldflags", "-s -w -X main.version=#{version}", "-trimpath", "-o", bin/"antibody"
+    system "go", "build", "-ldflags", "-X main.version=#{version}", "-trimpath", "-o", bin/"antibody"
```

Comment:
We don't use `std_go_args` here, so need to move to a separate PR (with adding `std_go_args` itself)

Reply:
good catch, I will revert both changes